src/lib/utils/featureFlags.ts

```diff
@@ -0,0 +1,33 @@
+export enum Environments {
+	TEST = 'test',
+	DEVELOPMENT = 'development',
+	PRODUCTION = 'production',
+}
+
+const isEnvironmentTest = (environment: string) =>
+	environment === Environments.TEST;
```

Comment:
@domw30  Can we get rid of this file as we're no longer using it?

Reply:
Yeah sure @padala, I will remove now and ping you once done

Reply:
Probably associated tests too.

Reply:
@domw30  Approved the PR but I think @colbr  should also approve it as he requested changes earlier on this.
Thanks